.github/workflows/lint.yaml

```
@@ -23,6 +23,9 @@ jobs:
         # check make init command does not fail
         run: make init project_id=PROJECT_ID builder_token=BUILDER_TOKEN output=/tmp/init-test
 
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v3
```

Sorry I have the additional comment.
WANT: I want fix terraform version to avoid the accident by upgrading the version.
```suggestion
      - name: Setup Terraform
        uses: hashicorp/setup-terraform@v3
        with:
          terraform_version: "1.11.3"
```

---

done!